.gitignore

```diff
@@ -152,3 +152,5 @@ results/
 
 # Slurm outputs
 slurm_logs/
```

Comment:
do we need this one now?

Reply:
Not anymore - I was still referring back to some previous logs there at one point but there's now enough new ones to get rid of this. Have removed it now